jsons/1.11.2.json

```diff
@@ -1,10 +1,10 @@
 {
     "assetIndex": {
```

Comment:
All changes to this file where generated by gradle. Let me know if I should revert.